env-defaults

```diff
@@ -188,12 +188,12 @@ SERVICE_RUNNER_LIMITS_MEMORY=512Mi
 SERVICE_RUNNER_IMAGE=harmonyservices/service-runner:latest
 
 # backend services
-ASF_GDAL_SUBSETTER_IMAGE=asfdataservices/gdal-subsetter:latest
-ASF_GDAL_SUBSETTER_REQUESTS_CPU=128m
-ASF_GDAL_SUBSETTER_REQUESTS_MEMORY=128Mi
-ASF_GDAL_SUBSETTER_LIMITS_CPU=128m
-ASF_GDAL_SUBSETTER_LIMITS_MEMORY=8Gi
-ASF_GDAL_SUBSETTER_INVOCATION_ARGS='python -m gdal_subsetter'
+HARMONY_GDAL_ADAPTER_IMAGE=nasa/harmony-gdal-adapter:latest
```

Comment:
I think this might need to be `ghcr.io/nasa/harmony-gdal-adapter:latest` looking at [an equivalent entry from PO.DAAC](https://github.com/nasa/harmony/blob/main/env-defaults#L247).

Reply:
ghcr.io/nasa/harmony-gdal-adapter:**sit  ????**